src/odemis/gui/comp/stream.py

```diff
@@ -634,6 +645,32 @@ def on_peak_btn(self, evt):
         event = stream_peak_event(state=self._header.btn_peak.GetState())
         wx.PostEvent(self, event)
 
+    @staticmethod
+    def create_text_frame(heading, text):
+        """
+        Create text frame with cancel button (same style as log frame during backend startup)
+        heading (String): title of the text box
+        text (String): text to be displayed
+        """
+        frame = wx.Dialog(None, title=heading, size=(800, 800),
+                          style=wx.DEFAULT_DIALOG_STYLE | wx.RESIZE_BORDER)
+        text = wx.TextCtrl(frame, value=text, style=wx.TE_MULTILINE | wx.TE_READONLY)
+
+        textsizer = wx.BoxSizer()
+        textsizer.Add(text, 1, flag=wx.ALL | wx.EXPAND)
+
+        btnsizer = frame.CreateButtonSizer(wx.CLOSE)
+
+        sizer = wx.BoxSizer(wx.VERTICAL)
+        sizer.Add(textsizer, 1, flag=wx.ALL | wx.EXPAND, border=5)
+        sizer.Add(btnsizer, 0, flag=wx.ALIGN_CENTER_VERTICAL | wx.EXPAND | wx.BOTTOM, border=5)
+        frame.SetSizer(sizer)
+
+        # Show the end of the log (which is most likely showing the error)
```

Comment:
Showing the end of the text by default shouldn't be needed here.
=> Drop the line (and the comment).